e2e/node/acp-v3.test.ts

```diff
@@ -48,6 +48,9 @@ import {
   TestingEnvironment,
 } from "../util/getTestingEnvironment";
 import { getAuthenticatedSession } from "../util/getAuthenticatedSession";
+import { setupTestResources, teardownTestResources } from "./test-setup";
```

Comment:
Can this be done via Jest?

Reply:
I'm not sure what you mean: the actual setup/teardown is done via jest's `beforeEach` and `afterEach`, the `test-setup` module just exports convenience functions to create resources and clear them up. It's just that it is shared across multiple test suites.

Reply:
Perhaps we'd be better calling it `test-helpers` instead of `test-setup` which is commonly used to refer to the jest setup?